server/lib/nl/detection/variable.py

```diff
@@ -49,6 +42,20 @@
 #
 def detect_vars(orig_query: str, debug_logs: Dict,
                 dargs: DetectionArgs) -> vars.VarDetectionResult:
+
+  # Get the list of stop words to use depending on if this is toolformer mode
+  # or not.
+  #
+  # TODO: decouple words removal from detected attributes. Today, the removal
+  # blanket removes anything that matches, including the various attribute/
+  # classification triggers and contained_in place types (and their plurals).
+  # This may not always be the best thing to do.
```

Comment:
Move this to `combine_stop_words` function def site?

Reply:
done